packages/mdc-slider/mdc-slider.scss

```diff
@@ -17,29 +17,33 @@
 @import "@material/theme/mixins";
 @import "@material/typography/mixins";
 @import "@material/rtl/mixins";
+@import "./keyframes";
+@import "./mixins";
 @import "./variables";
 
-$mdc-slider-main-grey: #bdbdbd;
-$mdc-slider-dark-theme-grey: #5c5c5c;
-
 // postcss-bem-linter: define slider
 
-@keyframes mdc-slider-emphasize {
-  0% {
-    animation-timing-function: ease-out;
-  }
+.mdc-slider {
+  @include mdc-slider-rail-color(secondary);
```

Comment:
Can this technically use the new higher-level `accessible` mixin? (Since IIUC `text-primary-on-secondary` should be the same value that the mixin computes when passed `secondary`?)

Reply:
We certainly could, but IIRC @lynnjepsen wanted our baseline component styles to explicitly call every individual theme mixin so that users know what all the options are. I'll verify this with her offline.

Reply:
I was wrong 😄   Lynn prefers your approach of using the high-level `accessible` mixin. Done!